mkroosevelt.js

```diff
@@ -2,32 +2,98 @@
   console.log("");
-  console.log("create an app in this directory:");
+  console.log("create an app in the current directory:");
+  console.log("mkroosevelt .");
+  console.log("");
+  console.log("create an app in this directory in a new folder:");
   console.log("mkroosevelt appName");
   console.log("");
   console.log("create an app somewhere else:");
   console.log("mkroosevelt /path/to/appName");
 }
 
+// Function to check the sampleApps files and then compare them against the current directories files to see if there is a match. If there is a match we handle it and ask the user if they want to continue and overwrite their files.
+function checkCurrentDirectoryForExistingFiles() {
+  // Set cmd to the current directory, so that we can create the app in the current directory
+  cmd = path.normalize(process.cwd());
+
+  // Walk the sampleApp files, so that we can match later against the walk of the current directory
+  fse.walk(path.normalize(__dirname + '/sampleApp/'))
+    .on('data', function (item) {
+
+      // Split on sampleApp so that we only get the file names and folders in the sampleApp not the absoulte path
+      item = item.path.toString().split('sampleApp');
+      sampleAppFiles.push(item[1])
+    })
+    // On end of the sampleApp's walk we start walking the current directory
+    .on('end', function () {
+
+      // Walk the current directories files, so that we can match against the walk we did of the sampleApp's directory and see if there is a match
+      fse.walk(path.normalize(cmd)) 
+        .on('data', function (item) {
+
+          // Split on the current directory so that we only get the file names and folders in the current directory not the absoulte path
+          item = item.path.toString().split(path.normalize(cmd));
+          directoryFiles.push(item[1]);
+        })
+        .on('end', function () {
+
+          // Delete the first element from each array as it is blank and will give a false positive on a conflict
+          directoryFiles.shift();
+          sampleAppFiles.shift();
+
+          // Loop through both the directory files and the sampleApp Files to see if there is a conflict
+          directoryFiles.forEach(function(element) {
+            sampleAppFiles.forEach(function(element1) {
+              if (element === element1) {
+                  conflictedFiles.push(element);
+              }
+            });
+          });
+
+          // If there any conflicting file(s) we log the file, so the user can see the file causing the conflict and ask them if they want to overwrite the file
+          if (conflictedFiles.length > 0) {
+            console.log('Your current directory contains files from the sample app')
+            conflictedFiles.forEach(function(file) {
+              console.log(file);
+            })
+            console.log("");
+            stdio.question('Continuing will overwrite these files. Would you like to continue? (y/n)', function (err, decision) {
```

Comment:
Yep. It is a common convention to imply that the no is the default answer and that you can just press enter and provide no input and that will be the same as entering no.